samples/lightweight-component/calc_pipeline.py

```diff
@@ -57,18 +57,18 @@ def divmod_helper(dividend, divisor):
 
 import kfp.dsl as dsl
 @dsl.pipeline(
-   name='Calculation pipeline',
+   name='calculation-pipeline',
    description='A toy pipeline that performs arithmetic calculations.'
 )
 # Currently kfp-tekton doesn't support pass parameter to the pipelinerun yet, so we hard code the number here
 def calc_pipeline(
-   a='7',
-   b='8',
-   c='17',
+   a:float = '7',
```

Comment:
nit: because its data type is float, I guess removing the quote would be more intuitive.

Reply:
thanks, updated